src/common.jl

```diff
@@ -14,3 +14,80 @@ test_dicts(match::Function, a::Dict, b::Dict) =
             end
         end
     end
+
+"""
+$(TYPEDSIGNATURES)
+
+Internal helper function that determines if a reaction has a gene reaction rule
+and that each gene in the rule is contained in the model.
+"""
+function _has_sensible_gpr(model, rid)
+    grrs = reaction_gene_association(model, rid)
+    isnothing(grrs) && return false
+    isempty(grrs) && return false
+    any("" in grr for grr in grrs) && return false
```

Comment:
```suggestion
    any(empty.(grrs)) && return false
```

Reply:
so this isn't looking for empty, instead for actual "" that might have snuck into the grrs... we really need to do some input validation to ensure that either a valid grr is input or nothing

Reply:
However, I also added that check

Reply:
ah it has one extra level.. ok np. a slightly more semantic way would then be:

```
any(isempty(x) for grr=grrs for g=grr)
```